lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -1046,6 +1055,7 @@ def _load_installed_snaps(self) -> None:
                 state=SnapState.Latest,
                 channel=i["channel"],
                 revision=i["revision"],
+                version=i["version"],
```

Comment:
Can we be sure that `"version"` is provided for all snaps? (e.g. [here](https://askubuntu.com/questions/915160/how-do-i-get-the-version-number-of-a-snap-package) is discussion of a snap where this field wasn't exposed).
```suggestion
                version=i.get("version"),
```